docs: apply tone guide to Responses.md (#1767)
carbon-design-system/carbon-ai-chat: packages/ai-chat/docs/Responses.md

}
```

> The `user_defined_type` field is a unique name for each UI component. Your renderer switches on it to decide what to draw. The `user_defined` response injects into a slot in the chat's shadow DOM. Style it from global CSS. It also inherits some styling from the chat, such as fonts. You can use Carbon components alongside your own components.

Reviewer: ```suggestion
The `user_defined_type` field is a unique name for each UI component. Your renderer switches on it to decide what to draw. The `user_defined` response injects into a slot within the chat's shadow DOM. You can style it from global CSS, and it inherits some styling from the chat, such as fonts. You can use Carbon components alongside your own custom components.
```

I prefer the old sentence better

### Streaming and updates

Prefer {@link ChatInstanceMessaging.upsertMessage | upsertMessage} to insert, stream, correct, and regenerate `user_defined` responses. One method covers all four. It drives a streaming UI from any source: SSE, WebSocket, polling, or whole-message snapshots. It also preserves component identity across updates. You accumulate state in your app and apply it with one call per update. This skips the chunk-shape contract below. It is the recommended direction for new code, but experimental. Its semantics and updater signature may still evolve. See [Adding messages (experimental)](./UpsertMessage.md). The chat supports nested `user_defined` items inside {@link MessageResponseTypes.CARD | card}, {@link MessageResponseTypes.CAROUSEL | carousel}, and {@link MessageResponseTypes.GRID | grid} containers.

{@link ChatInstanceMessaging.addMessageChunk | addMessageChunk} is the stable streaming path. It is fully supported with no deprecation, and the right choice if you prefer a settled API. If you stream with it, read [Adding messages (legacy)](./AddMessageChunk.md) first. The model is the same in every framework:

> - {@link RenderUserDefinedState.partialItems | `partialItems`} is an array of every chunk received, **not** concatenated for you. The streaming API sends string chunks, not partial JSON. So stringify your JSON. Then concatenate and parse the chunks in your renderer with `try`/`catch` or an optimistic parser.

Reviewer: ```suggestion
- {@link RenderUserDefinedState.partialItems | `partialItems`} is an array of every chunk received, **not** concatenated for you. The streaming API sends string chunks, not partial JSON. So stringify your JSON, then concatenate and parse the chunks in your renderer with `try`/`catch` or an optimistic parser.
```
